### PR TITLE
Add simplified printing press

### PR DESCRIPTION
**Pull request by waffle-stomper**
[Maester Flaminius](https://www.reddit.com/user/flaminius) has asked me to add these recipes to the printing press. I'll work on the more advanced features once I've wrapped my head around them.

### CODE REVIEW

File: src/main/resources/configCivcraft.yml [hunk 1]
```diff
@@ -1543,6 +1545,39 @@ recipes:
...
+    input:
+      leather:
+        material: LEATHER
+        amount: 6
+      paper:
+        material: PAPER
+        amount: 18
+    output:
+      books:
+        material: BOOK
```
Comment (Maxopoly) [inline src/main/resources/configCivcraft.yml]: This will produce empty, possibly unusable (due to missing nbt) written books, so kinda pointless

Comment (waffle-stomper) [inline src/main/resources/configCivcraft.yml]: I'm confused. It seems to produce [these books](http://minecraft.gamepedia.com/Book) as intended when I run it on my test server. I can successfully use them to create bookshelves and book & quill books, and they refuse to open with a right click.

I thought written books were org.bukkit.Material.WRITTEN_BOOK?

Comment (Maxopoly) [inline src/main/resources/configCivcraft.yml]: Wait yeah, I havent played minecraft for too long. Nevermind.

Comment (waffle-stomper) [inline src/main/resources/configCivcraft.yml]: You should get back into it - it's a lot of fun, (if a bit of a time sink). There's this great server called Civcraft you should definitely check out.

Comment (ttk2) [inline src/main/resources/configCivcraft.yml]: so is this patch good and not broken?

Comment (Maxopoly) [inline src/main/resources/configCivcraft.yml]: I guess so. Should we just merge this and then merge it into gitlab or do the PR there?

**Pull request merged**